src/mutations/item.ts

```diff
@@ -337,9 +337,17 @@ export default (queryConfig: QueryClientConfig) => {
 
   const useCopyItems = () =>
     useMutation(
-      ({ ids, to }: { ids: UUID[]; to?: UUID }) =>
+      ({
+        ids,
+        to,
+        newNames,
+      }: {
+        ids: UUID[];
+        to?: UUID;
+        newNames?: { [key: string]: string };
```

Comment:
I know UUID is an alias for string but if we use it here it will hint towards the fact that the keys should be ids and not any random strings.
```suggestion
        newNames?: { [key: UUID]: string };
```